### PR TITLE
Added BQ500100DCK current shunt amplifier

### PR DESCRIPTION
**Pull request by SchrodingersGat**
Added symbol fro BQ500100DCK current sense amplifier in SC-70-6 package

https://www.digikey.com/product-detail/en/texas-instruments/BQ500100DCKT/296-48499-1-ND/8567543

- Copied from NCS210 symbol
- Datasheet: http://www.ti.com/lit/ds/symlink/bq500100.pdf

image

image

---

All contributions to the kicad library must follow the [KiCad library convention](http://kicad-pcb.org/libraries/klc/)

Thanks for creating a pull request to contribute to the KiCad libraries! To speed up integration of your PR, please check the following items:

- [x] Provide a URL to a datasheet for the symbol(s) you are contributing
- [x] An example screenshot image is very helpful
- [x] Ensure that the associated footprints match the [official footprint library](https://github.com/kicad/kicad-footprints)
  - A new fitting footprint must be submitted if the library does not yet contain one.
- [ ] If there are matching footprint PRs, provide link(s) as appropriate
- [ ] Check the output of the Travis automated check scripts - fix any errors as required
- [ ] Give a reason behind any intentional library convention rule violation.

### CODE REVIEW

File: Amplifier_Current.lib [hunk 1]
```diff
@@ -688,7 +709,7 @@ $FPLIST
 $ENDFPLIST
 DRAW
 S -300 300 400 -200 0 1 10 f
-X REF 1 500 0 100 L 50 50 1 1 I
+X GND 1 100 -300 100 U 50 50 1 1 W
```
Comment (fauxpark) [inline Amplifier_Current.lib]: I don't think you meant to change this pin :)

Comment (SchrodingersGat) [reply Amplifier_Current.lib]: I don't think I did either.. Thanks

**Comment by SchrodingersGat** [comment]
@fauxpark fixed the error, thanks for spotting

**Comment by myfreescalewebpage** [comment]
Hi @SchrodingersGat , thanks for contributing,

A few comments I have during my review:

- [ ] The design of the symbol is not really nice, I think something like the INA196, looking an op amp, is clearly better
- [ ] Datasheet link is not correct
- [ ] Double comma in the description after Voltage Output, to be fixed
- [ ] Footprint filter should be `SOT?363*`
- [ ] Name of pin 3 is `V+`

Cheers,
Joel

**Comment by SchrodingersGat** [comment]
image

Thanks for the review @myfreescalewebpage changes have been made as requested.

**Comment by myfreescalewebpage** [comment]
@SchrodingersGat thanks. Error in the footprint filter, the leading star is not required. Can you fix it, then we can merge?

**Comment by SchrodingersGat** [comment]
Done

**Comment by myfreescalewebpage** [comment]
Thanks, merging.

**Pull request merged**